source/lambda/apiprocessor/lambda_function.py

```diff
@@ -81,26 +106,38 @@ def lambda_handler(event, context):
                     for obj in body['objects']:
                         request["bucketName"] = sampleBucket if obj['sample'] else documentBucket
                         request["objectName"] = obj['key']
-                        results.append(createDocument(request))
+                        if validate_request(request):
+                            results.append(createDocument(request))
+                        else:
+                            status_code = 207
```

Comment:
Should this be a 207 code or a 400 error since its client side?

Reply:
207 is if some of the documents create successfully, but others fail.